packages/react/src/components/UIShell/index.js

```diff
@@ -5,35 +5,35 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-export Content from './Content';
+export { default as Content } from './Content';
 
-export Header from './Header';
-export HeaderContainer from './HeaderContainer';
-export HeaderGlobalAction from './HeaderGlobalAction';
-export HeaderGlobalBar from './HeaderGlobalBar';
-export HeaderMenu from './HeaderMenu';
-export HeaderMenuButton from './HeaderMenuButton';
-export HeaderMenuItem from './HeaderMenuItem';
-export HeaderName from './HeaderName';
+export { default as Header } from './Header';
+export { default as HeaderContainer } from './HeaderContainer';
+export { default as HeaderGlobalAction } from './HeaderGlobalAction';
+export { default as HeaderGlobalBar } from './HeaderGlobalBar';
+export { default as HeaderMenu } from './HeaderMenu';
+export { default as HeaderMenuButton } from './HeaderMenuButton';
+export { default as HeaderMenuItem } from './HeaderMenuItem';
+export { default as HeaderName } from './HeaderName';
 export { HeaderNavigation } from './HeaderNavigation';
-export HeaderPanel from './HeaderPanel';
-export HeaderSideNavItems from './HeaderSideNavItems';
-export Switcher from './Switcher';
-export SwitcherItem from './SwitcherItem';
-export SwitcherDivider from './SwitcherDivider';
+export { default as HeaderPanel } from './HeaderPanel';
```

Comment:
what are your thoughts about also exporting `HeaderNavigation` and `SideNavMenu` as default exports in their respective files so that they can follow the same paradigm here in this file?